Analysis/DEEP2_R23_O32/calibration_plots.py

```diff
@@ -100,88 +101,73 @@ def LAC_GPC_plots(fitspath, dataset,revised= False, individual = False, include_
-        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, ctype=['b','r','m'], label=label, silent=False, verbose=True)
+        lR23 = [det_R23, der_R23, der_R23_MACT]
+        lO32 = [det_O32, der_O32, der_O32_MACT]
+        OH = [det_OH, der_OH, der_OH_MACT]
+        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, ctype=['b', 'r', 'm'], label=label,
+                                      silent=False, verbose=True)
         
     if dataset == 'O32_Grid' or dataset == 'Grid':    
-        lR23 = [det_R23,rlimit_R23,der_R23,der_R23_MACT]
-        lO32 = [det_O32,rlimit_O32,der_O32,der_O32_MACT]
-        OH   = [det_OH,rlimit_OH, der_OH, der_OH_MACT]
-        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, ctype=['b','g','r','m'], label=['Detection','Non-Dectection','DEEP2', 'MACT'], silent=False, verbose=True)
-    # ID=[det_ID,nandet_ID]
+        lR23 = [det_R23, rlimit_R23, der_R23, der_R23_MACT]
+        lO32 = [det_O32, rlimit_O32, der_O32, der_O32_MACT]
+        OH   = [det_OH, rlimit_OH, der_OH, der_OH_MACT]
+        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, ctype=['b', 'g', 'r', 'm'],
+                                      label=['Detection', 'Non-Dectection,'DEEP2', 'MACT'], silent=False, verbose=True)
 
     if dataset == 'Voronoi10' or dataset == 'Voronoi14' or dataset == 'Voronoi20' or dataset =='Double_Bin':
-        lR23 = [det_R23,rlimit_R23,der_R23,der_R23_MACT]
-        lO32 = [det_O32,rlimit_O32,der_O32,der_O32_MACT]
-        OH   = [det_OH,rlimit_OH, der_OH, der_OH_MACT]
+        lR23 = [det_R23, rlimit_R23, der_R23, der_R23_MACT]
+        lO32 = [det_O32, rlimit_O32, der_O32, der_O32_MACT]
+        OH   = [det_OH, rlimit_OH, der_OH, der_OH_MACT]
 
-        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0,9.0], ctype=['b','g','r','m'], label=['Detection','Non-Dectection','DEEP2','MACT'], silent=False, verbose=True)
+        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=['b', 'g', 'r', 'm'],
+                                      label=['Detection', 'Non-Dectection', 'DEEP2', 'MACT'], silent=False, verbose=True)
 
     if dataset == 'n_Bins':
-        lR23 = [det_R23,rlimit_R23,der_R23,der_R23_MACT]
-        lO32 = [det_O32,rlimit_O32,der_O32,der_O32_MACT]
-        OH   = [det_OH,rlimit_OH, der_OH, der_OH_MACT]
+        lR23 = [det_R23, rlimit_R23, der_R23, der_R23_MACT]
+        lO32 = [det_O32, rlimit_O32, der_O32, der_O32_MACT]
+        OH   = [det_OH, rlimit_OH, der_OH, der_OH_MACT]
 
-        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0,9.0], ctype=['b','g','r','m'], label=['Detection','Non-Dectection','DEEP2','MACT'], silent=False, verbose=True)
+        local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=['b', 'g', 'r', 'm'],
+                                      label=['Detection', 'Non-Dectection', 'DEEP2', 'MACT'], silent=False, verbose=True)
```

Comment:
I did it this way because the ctype is different for different cases. However, we haven't touched the other datasets for quite some time. Do you want me to keep the other datasets as an option?